recipes/rmblast/meta.yaml

```diff
@@ -12,37 +12,36 @@ source:
   patches:
     #- isb-{{ version }}+-{{ name }}.patch # Needs the -p1 option, applied in build.sh
     # patch taken from http://www.repeatmasker.org/rmblast/isb-2.14.0+-rmblast.patch.gz
+    - isb-2.14.0+-rmblast.patch
     - configurellvm.patch
     - get_species_taxids.patch
     - phonehome.patch
 
 build:
-  number: 1
-  skip: True # [osx]
+  number: 2
+  ignore_run_exports:
```

Comment:
Why is this needed?

Reply:
It is not, I have removed ignore_run_exports in my next commit.